README.md

```diff
@@ -1,32 +1,56 @@
 
-![Crayfish](https://user-images.githubusercontent.com/2371345/48163075-11c6cf80-e2b5-11e8-8b5b-991b366014a5.png) 
+![Crayfish](https://user-images.githubusercontent.com/2371345/48163075-11c6cf80-e2b5-11e8-8b5b-991b366014a5.png)
 # Crayfish Commons
 
-[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%205.6-8892BF.svg?style=flat-square)](https://php.net/)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.2-8892BF.svg?style=flat-square)](https://php.net/)
 [![Build Status](https://travis-ci.com/Islandora-CLAW/Crayfish-Commons.svg?branch=master)](https://travis-ci.com/Islandora-CLAW/Crayfish-Commons)
 [![Contribution Guidelines](http://img.shields.io/badge/CONTRIBUTING-Guidelines-blue.svg)](./CONTRIBUTING.md)
 [![LICENSE](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](./LICENSE)
 [![codecov](https://codecov.io/gh/Islandora-CLAW/Crayfish-Commons/branch/master/graph/badge.svg)](https://codecov.io/gh/Islandora-CLAW/Crayfish-Commons)
 
+
+## Introduction
+
 Crayfish Commons is a library housing shared code for Crayfish microservices.
+Shared Classes include:
+* GeminiClient
+* IslandoraServiceProvider
+* YamlConfigServiceProvider
+* JwtAuthenticator
+* JwtFactory
+* JwtUser
+* SettingsParser
+* ApixMiddleware
+* CmdExecuteService
 
 ## Requirements
 
-* PHP 5.6+
+* PHP 7.2+
 * [Composer](https://getcomposer.org/)
 
-## Development
+## Installation
+Crayfish-commons can be installed with composer like `composer require islandora/crayfish-commons`
+
+## Configuration
+There is no related configuration as it serves as a dependency for  microservices.
 
-If you would like to contribute, please get involved by attending our weekly 
-[Tech Call][5]. We love to hear from you!
+## Documentation
 
-If you would like to contribute code to the project, you need to be covered by 
-an Islandora Foundation [Contributor License Agreement][6] or 
-[Corporate Contributor License Agreement][7]. Please see the 
-[Contributors][8] pages on Islandora.ca for more information.
+Further documentation for this module is available on the [Islandora 8 documentation site](https://islandora-claw.github.io/CLAW/).
```

Comment:
https://github.com/Islandora/documentation/tree/master/docs

Reply:
this should be fixed now